app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class DishesController < ApplicationController
+  def show
+    @dish = Dish.find(params[:dish_id])
+    @chef = @dish.chef
```

Comment:
Keep in mind that you can call `@dish.chef` and `@disch.kcal_sum` in the view so these extra instance vars are not strictly necessary!